tests/local-tests/index.js

```diff
@@ -36,6 +37,7 @@ const testComponents = {
   'Firestore onSnapshotInSync Test': FirestoreOnSnapshotInSyncTest,
   'VertexAI Generation Example': VertexAITestComponent,
   'Auth MFA Demonstrator': AuthMFADemonstrator,
+  'HttpsCallable Test': HttpsCallableTestComponent,
```

Comment:
Honestly, just makes me a little happy to see the local test infra used for a quick otherwise-one-off test that still may have value in future, but mostly you just need some way to run a little thing fast and here it is